src/ofxOssia.cpp

```diff
@@ -24,6 +31,31 @@ void ofxOssia::setup(const std::string& type,
     _root_node.setup (_device.getRootNode(), "ossia");
 }
 
+void explore(const ossia::net::node_base& node)
+{
+  for (const ossia::net::node_base* child : node.children_copy())
+  {
+    if (auto addr = child->getAddress())
+    {
+	std::cerr << ossia::net::osc_address_string(*addr) << " : "
+		  << ossia::value_to_pretty_string(addr->fetchValue()) << "\n";
+	addr->pullValue();
+
+    }
+    explore(*child);
+  }
+}
+
+using namespace ossia::oscquery;
+
+void ofxOssia::setup(const std::string& serverName, int serverPortWS, int localPortOSC){
+     auto& local_proto = static_cast<ossia::net::local_protocol&>(_device.getProtocol());
```

Comment:
a general remark (not only about this commit but also the general code): couldn't the local_proto be put as a reference in the object upon construction, since it will be accessed almost everywhere anyway ?